### PR TITLE
COMMERCE-4052 Make commerce deploy on master

### PR DESCRIPTION
**Pull request by dsitu**

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by liferay-commerce** [comment]
ci:test:sf

**Comment by liferay-commerce** [comment]
ci:test:sf

File: modules/suites/commerce/suite.bnd [hunk 1]
```diff
@@ -0,0 +1,2 @@
... with other users. The suite's social networking apps and features enable you to attribute social equity to active users automatically by increasing their rank on the site. You can even highlight user activities to show your site's liveliness. The Mentions feature facilitates notifying specific users in the context of site content, discussions, and comments. The social apps also help users share their thoughts on the your sites and other social sites, such as Facebook and Twitter. Combined with the suite's collaboration and document management features, the Collaboration suite's social features help develop awesome communities, foster high morale, and make work interesting.<br /><br />Apps and Features: Activities, Alerts, Announcements, Blogs, Bookmarks, Comments, Friends/My Sites/Portal/Site Members Directory, Documents and Media, Friend Requests, Image Uploader, Invite Members, Media Gallery, Message Boards, Mentions, Microblogs, Page Flags/Comments/Ratings, Quick Note, Requests, Social Activity, User/Group Statistics, Wiki.<br /><br />This suite may contain modules that depend on Foundation suite modules. This suite is available in the ${liferay.releng.app.title.prefix} bundle.
```
Comment (liferay-commerce) [inline modules/suites/commerce/suite.bnd]: @dsitu maybe we need to find a better commerce text, and ask Corbin

Comment (dsitu) [reply modules/suites/commerce/suite.bnd]: oh yeah, opps ahha, I thought I had removed that description...

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by liferay-commerce** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by pyoo47** [comment]
ci:test:sf:norebase

**Comment by liferay-commerce** [comment]
Merged in COMMERCE-4052

**Pull request closed without merging**